plugins/email-messages/impl/src/test/java/org/pentaho/di/job/entries/getpop/JobEntryGetPOPTest.java

```diff
@@ -320,4 +337,37 @@ public void testEnvVariablesAreSubstitutedForFolders() {
       fail( "Unexpected exception when calling createOutputDirectory for attachment directory" );
     }
   }
+  @Test
+  public void testgrantTypeIsClientCredentials() {
+   entry.setGrant_type( JobEntryGetPOP.GRANTTYPE_CLIENTCREDENTIALS );
+    assertEquals( JobEntryGetPOP.GRANTTYPE_CLIENTCREDENTIALS, entry.getGrant_type() );
+  }
+
+  @Test
+  public void testgrantTypeIsAuthorizationCode() {
+    entry.setGrant_type( JobEntryGetPOP.GRANTTYPE_AUTHORIZATION_CODE );
+    assertEquals( JobEntryGetPOP.GRANTTYPE_AUTHORIZATION_CODE, entry.getGrant_type() );
+  }
+
+  @Test( expected = RuntimeException.class )
```

Comment:
Can we test for something more selective than a RuntimeException?  A lot of things could possibly cause this test to look like it works when it did not.